src/main/java/roomescape/service/ReservationService.java

```diff
@@ -99,18 +122,48 @@ public List<ReservationResponse> findAll() {
     public List<ReservationResponse> searchReservation(Long themeId, Long memberId, LocalDate dateFrom,
                                                        LocalDate dateTo) {
         Reservations reservations = new Reservations(reservationRepository.findByThemeIdAndMemberIdAndDateBetween(themeId, memberId, dateFrom, dateTo));
-        return reservations.getReservations().stream()
+        return reservations.booked().stream()
                 .map(ReservationResponse::from)
                 .toList();
     }
 
-    public void delete(long reservationId) {
+    @Transactional
+    public List<ReservationDetailResponse> findAllByMemberId(long memberId) {
+        Reservations reservations = new Reservations(reservationRepository.findAllByMemberId(memberId));
+        List<Waiting> waitings = reservations.waiting().stream()
+                .map(reservation -> new Waiting(reservation,
+                        reservationRepository.findAndCountWaitingNumber(
+                                reservation.getDate(),
+                                reservation.getReservationTime(),
+                                reservation.getTheme(),
+                                reservation.getCreatedAt())))
+                .filter(waiting -> !waiting.isOver())
+                .toList();
+        return ReservationDetailResponse.of(reservations, waitings);
+    }
+
+    @Transactional
+    public void deleteById(long reservationId) {
         reservationRepository.deleteById(reservationId);
     }
 
-    public List<ReservationDetailResponse> findAllByMemberId(long userId) {
-        return reservationRepository.findAllByMemberId(userId).stream()
-                .map(ReservationDetailResponse::from)
+    @Transactional
+    public void deleteByMemberIdAndId(LoginMember loginMember, long id) {
+        reservationRepository.deleteByMemberIdAndId(loginMember.getId(), id);
+    }
+
+    @Transactional
+    public List<AdminReservationDetailResponse> findAllWaitingReservations() {
+        List<Reservation> reservations = reservationRepository.findAllByStatus(ReservationStatus.WAITING);
+        return reservations.stream()
+                .map(reservation ->
+                        new Waiting(reservation, reservationRepository.findAndCountWaitingNumber(
+                                reservation.getDate(),
+                                reservation.getReservationTime(),
+                                reservation.getTheme(),
+                                reservation.getCreatedAt())))
```

Comment:
```suggestion
                        new Waiting(
                            reservation,
                            reservationRepository.findAndCountWaitingNumber(
                                reservation.getDate(),
                                reservation.getReservationTime(),
                                reservation.getTheme(),
                                reservation.getCreatedAt())))
```

조금 더 명확하게 어떤 파라미터인지 구분해보는건 어떨까요?